keel-core/src/test/kotlin/com/netflix/spinnaker/keel/services/ResourceStatusServiceTests.kt

```diff
@@ -30,238 +29,213 @@ import com.netflix.spinnaker.keel.persistence.ResourceStatus.MISSING_DEPENDENCY
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.PAUSED
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.RESUMED
 import com.netflix.spinnaker.keel.persistence.ResourceStatus.UNHAPPY
-import com.netflix.spinnaker.keel.test.combinedInMemoryRepository
-import com.netflix.spinnaker.keel.test.deliveryConfig
 import com.netflix.spinnaker.keel.test.resource
 import com.netflix.spinnaker.kork.exceptions.SpinnakerException
-import com.netflix.spinnaker.time.MutableClock
 import dev.minutest.junit.JUnit5Minutests
 import dev.minutest.rootContext
 import io.mockk.every
-import io.mockk.slot
+import io.mockk.mockk
 import strikt.api.expectThat
 import strikt.assertions.isEqualTo
 
 class ResourceStatusServiceTests : JUnit5Minutests {
   companion object Fixture {
-    val clock = MutableClock()
-    val repository = combinedInMemoryRepository(clock)
-    val actuationPauser = ActuationPauser(repository.resourceRepository, repository.pausedRepository, repository.publisher, clock)
-    val subject = ResourceStatusService(repository.resourceRepository, actuationPauser)
+    val repository = mockk<ResourceRepository>()
+    val actuationPauser = mockk<ActuationPauser>()
+    val subject = ResourceStatusService(repository, actuationPauser)
```

Comment:
I thought it made more sense to stub `ActuationPauser` rather than testing two levels at once. `ActuationPauser` has its own tests.